### ISSUE TITLE
[Dragon Realm] Forestry backpack configs appear unset

### ISSUE THREAD

**Issue by Gronglegrowth** [ReikaKalseki/Reika_Mods_Issues#3226]
This may be from installation errors, but the forestry backpacks (miner's, forester's, etc etc) will not pick up items that I expect them to (ores going into miner's, logs and saplings in forester's, etc), and upon looking at the config file for them, they appear to be left at their defaults, which is only allowing certain items from mostly Railcraft in them. Is that intentional?

**Comment by ReikaKalseki** [comment]
No, but I have no idea what to do to make them work; I have never used these.

**Comment by Gronglegrowth** [comment]
In the config for the backpacks, you would specify the items and oredict tags that should be storable in each type; the existing values there are probably enough examples on the expected format.

I was considering spending time setting their configs myself, as I like using the backpacks when they are available.

**Comment by ReikaKalseki** [comment]
This looks like a pain to configure...I almost want to use DRCore to populate them reflectively.

**Comment by Jkysler** [comment]
I have configured them manually myself. It is not very difficult. There is oredict tagging and smoothens the process. I can share my config if desired.

**Comment by Jkysler** [comment]
I'll just leave this here. edit: made a small change to add all seed oredict to forester backpack

[backpacks.zip](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/12065511/backpacks.zip)
